### DISCUSSION TITLE
Is there any guide or link on how to deploy wfb_ng on Android pad/phone?

### DISCUSSION THREAD

**Discussion by lida2003** [svpcom/wfb-ng#342] [Questions]
Anyone knows about Android device with kind of wifi card can deploy wfb_ng?

**Comment by svpcom** [comment]
The problem with android devices is that you need a root access for:
- Compiling and adding 8812au driver
- Running packet capture/injection

But there is a https://github.com/gehee/FPVue_android which use **userspace** driver (based on libusb) which support **rx-only mode** for 8812au.

**Reply by svpcom** [reply]
fpvue_android can only receive wfb-ng video stream and there is some initial support for receiving telemetry
